applications/StructuralMechanicsApplication/custom_elements/bushing_element.h

```diff
@@ -0,0 +1,332 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/element.h"
+#include "includes/ublas_interface.h"
+
+namespace Kratos
+{
+///@name Kratos Classes
+///@{
+
+class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) BushingElement: public Element
```

Comment:
can you please add some docs? It is not very clear what this does/how it should be used

Reply:
I will :)